README.md

```diff
@@ -20,6 +20,17 @@ docker preferences in the via the dock icon.
 * Disk (Ensure the file has type `raw`)
 * Advanced (CPU's, Memory)
 
+From version 2.x this tool supports the usage of NFS mounted name-volumes, which
+gives faster file synchronisation during composer install and cache clear etc.
+
+It comes with a command `nfs:enable` that exports the users (yours) home directory
```

Comment:
It doesn't export a home directory (unless you're using your Mac in a very peculiar way), but the `Users` directory.

Maybe just write

> … that exports the `Users` directory through …

Reply:
Fixed